app/assets/stylesheets/application.scss

```diff
@@ -23,6 +23,14 @@ body {
   font-weight: normal;
 }
 
+.button{
+  background-color: $primary-color;
+}
+
+.button.alert{
```

Comment:
Merge rule `.button.alert` with rule on line 26<br>Opening curly brace `{` should be preceded by one space